perllib/Open311/UpdatesBase.pm

```diff
@@ -287,24 +294,26 @@ sub comment_text_for_request {
             auto_response => 1,
             -or => $state_params,
         }, $order )->first) {
-            $template = $t->text;
+            $template = $t->text,
+            $template_email_text = $t->email_text,
         }
     }
 
+    ### NEED TO NOW SET THE TEST TO SET use_template_email ###
     my $desc = $request->{description} || '';
     if ($desc && (!$template || ($template !~ /\{\{description}}/ && !$request->{prefer_template}))) {
-        return $desc;
+        return ($desc, '');
     }
 
     if ($template) {
         $template =~ s/\{\{description}}/$desc/;
-        return $template;
+        return ($template, $template_email_text);
     }
 
-    return "" if $self->blank_updates_permitted;
+    return ("", "") if $self->blank_updates_permitted;
```

Comment:
Should this, and few lines down, return undef as second argument for consistency? I think that way it would always be NULL set if not present, rather than sometimes null and sometimes the empty string? If I've understood it right.

Reply:
Also applied a couple of lines up for same reason now